terraform/features-api/vars/dev.tf

```diff
@@ -0,0 +1,14 @@
+region             = "us-west-2"
+registry_name      = "veda-wfs3-registry"
+env                = "dev"
+project_name       = "veda-features-api"
+availability_zones = ["us-west-2a", "us-west-2b"]
+service_port       = 8080
+dns_zone_name      = "delta-backend.com"
+dns_subdomain      = "firenrt"
```

Comment:
as we talked about you'll probably want to change the `dns_subdomain` to `ghg` or something and check what `dns_zone_name` is available in MCP for ghg